packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -63,6 +64,31 @@ void main() {
     expect(widget.style.color.alpha, lessThan(255));
   });
 
+  testWidgets('Has semantic annotations', (WidgetTester tester) async {
+    final SemanticsTester semantics = new SemanticsTester(tester);
+    await tester.pumpWidget(MaterialApp(home: const Material(
+      child: CupertinoAlertDialog(
+        title: Text('The Title'),
+        content: Text('Content'),
+        actions: <Widget>[
+          CupertinoDialogAction(child: Text('Cancel')),
+          CupertinoDialogAction(child: Text('OK')),
+        ],
+      )
+    )));
+
+    expect(semantics, includesNodeWith(
+      label: 'Alert',
+      flags: <SemanticsFlag>[SemanticsFlag.namesRoute, SemanticsFlag.scopesRoute]
+    ));
+    expect(semantics, includesNodeWith(label: 'The Title'));
```

Comment:
If you're going to test the content of multiple nodes I would just use the hasSemantics matcher